modules/custom/az_migration/az_migration_http_cache/az_migration_http_cache.services.yml

```
@@ -0,0 +1,7 @@
+services:
+  cache.az_migration_http_cache:
+     class: Drupal\Core\Cache\CacheBackendInterface
+     tags:
+       - { name: cache.bin }
+     factory: cache_factory:get
+     arguments: [az_migration_http_cache]
```

ESlint is complaining about the whitespace in this file.
```suggestion
    class: Drupal\Core\Cache\CacheBackendInterface
    tags:
      - { name: cache.bin }
    factory: cache_factory:get
    arguments: [az_migration_http_cache]

```